src/pages/signin/signin.js

```diff
@@ -0,0 +1,74 @@
+import validateEmail from '../../components/EmailValidation.js';
+
+const emailInput = document.querySelector('.email');
+const errMessageEmail = document.querySelector('.email-valid');
+const pwdInput = document.querySelector('.password');
+const errMessagePwd = document.querySelector('.password-valid');
+const loginBtn = document.querySelector('.loginBtn');
+
+let isEmailValid,
+  isPwdValid = false;
+
+// 이메일 유효성 체크 함수
+export const emailCheck = event => {
+  const { value } = event.target;
+
+  if (!value.trim()) {
+    event.target.classList.add('error');
+    errMessageEmail.textContent = '이메일을 입력해주세요';
+    isEmailValid = false;
+    errMessageEmail.style.visibility = 'visible';
+    return isEmailValid;
+  }
+
+  if (!validateEmail(value)) {
+    event.target.classList.add('error');
+    errMessageEmail.textContent = '잘못된 이메일 형식입니다';
+    errMessageEmail.style.visibility = 'visible';
+    isEmailValid = false;
+    return isEmailValid;
+  }
+
+  event.target.classList.remove('error');
+  errMessageEmail.style.visibility = 'hidden';
+  isEmailValid = true;
+  updateBtnState();
+
+  return isEmailValid;
+};
+// 비밀번호 유효성 체크 함수
+export const pwdCheck = e => {
```

Comment:
```suggestion
export const pwdCheck = event => {
```
위에서 약어를 안써주셨으면 아래서도 안써주시는게 컨벤션적으로 좋을 것 같습니다!